engine/src/main/java/org/terasology/rendering/nui/internal/NUIManagerInternal.java

```diff
@@ -65,7 +70,7 @@
  */
 public class NUIManagerInternal extends BaseComponentSystem implements NUIManager {
     private Logger logger = LoggerFactory.getLogger(NUIManagerInternal.class);
-    private Deque<UIScreenLayer> screens = Queues.newArrayDeque();
+    private LinkedList<UIScreenLayer> screens = new LinkedList<>();
```

Comment:
Um, is that link wrong? I'm not seeing anything there that requires a linked list.